packages/flutter/lib/src/painting/image_provider.dart

```diff
@@ -17,6 +17,8 @@ import 'binding.dart';
 import 'image_cache.dart';
 import 'image_stream.dart';
 
+export '_network_image_io.dart';
```

Comment:
Instead of exporting here for the network image, define the exception class in image_provider.dart

Reply:
Done

Reply:
This line needs to be removedstill

Reply:
Yeah forgot about it, sorry. Fixed now.